include/flatbuffers/grpc.h

```diff
@@ -23,49 +23,233 @@
 #include "grpc++/support/byte_buffer.h"
 #include "grpc/byte_buffer_reader.h"
 
+namespace flatbuffers {
+namespace grpc {
+
+enum AddRef { ADD_REF };
+enum StealRef { STEAL_REF };
+
+// Message is a typed wrapper around a buffer that manages the underlying
+// `grpc_slice` and also provides flatbuffers-specific helpers such as `Verify`
+// and `GetRoot`.
```

Comment:
I think it would help to expand this doc comment and say that `Message`, like `grpc_slice`, has shared reference counted ownership of the underlying buffer. This is obvious if you know `grpc_slice` and read the code but it would still help I think.